migrator/migrations/m_212_to_m_213_add_container_start_column_to_indicators/migration_impl.go

```diff
@@ -26,8 +26,22 @@ func migrate(database *types.Databases) error {
 	pgutils.CreateTableFromModel(database.DBCtx, db, updatedSchema.CreateTableProcessIndicatorsStmt)
 	db = db.WithContext(database.DBCtx).Table(updatedSchema.ClustersTableName)
 
+	// Drop the indexes except the cluster one.
```

Comment:
readability nit: move to separate funcs? dropIndexes and reCreateIndexes?  Also a comment with the why would be a nice touch.